external/fv3kube/fv3kube/config.py

```diff
@@ -157,13 +157,24 @@ def get_full_config(
 
 
 def c48_initial_conditions_overlay(
-    url: str, timestep: str, restart_categories: RestartCategoriesConfig = None
+    url: str, timestep: str, vertical_coordinate_file: str, restart_categories: RestartCategoriesConfig = None
```

Comment:
Similar to what is done for `restart_categories`, I might make `vertical_coordinate_file` a keyword argument.  In this case I would use a default value of `DEFAULT_VERTICAL_COORDINATE_FILE`.  This way the [existing regression test](https://github.com/ai2cm/fv3net/blob/0deee7a220a4de6fb2d9fe807c77266def6ff956/external/fv3kube/tests/test_config.py#L35-L41), which does not pass `vertical_coordinate_file` or `restart_categories` arguments, would still pass.